internal/api/v1/organizations.go

```diff
@@ -161,3 +155,53 @@ func (oc OrganizationsController) Delete(w http.ResponseWriter, r *http.Request)
 
 	return nil
 }
+
+func deleteApps(cluster *kubernetes.Cluster, gitea *gitea.Client, org string) error {
+	apps, err := application.List(cluster, org)
+	if err != nil {
+		return err
+	}
+
+	const maxConcurrent = 100
+	errChan := make(chan error, maxConcurrent)
+	success := make(chan struct{})
+	stop := make(chan struct{})
```

Comment:
Since these structs have no fields, maybe using `bool` is better (cleaner code).

Reply:
done